From 1e0ea5fb708764bb5087043dd162409602744175 Mon Sep 17 00:00:00 2001
From: Pablo Fraile Alonso <pablo@pablofraile.net>
Date: Thu, 29 Feb 2024 20:17:56 +0100
Subject: [PATCH] feat: print timeout

---
 tcp/tcp.cc | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/tcp/tcp.cc b/tcp/tcp.cc
index a919c8c..2dcbed8 100644
--- a/tcp/tcp.cc
+++ b/tcp/tcp.cc
@@ -978,7 +978,10 @@ void TcpAgent::reset_rtx_timer(int mild, int backoff)
  */
 void TcpAgent::set_rtx_timer()
 {
-	rtx_timer_.resched(rtt_timeout());
+    double timeout = rtt_timeout();
+    double now = Scheduler::instance().clock();
+	rtx_timer_.resched(timeout);
+    fprintf(stderr, ""TIME_FORMAT" "TIME_FORMAT" \n", now, timeout);
 }
 
 /*
-- 
2.43.2

