From 82c0a69bb1e1034ec406bfe0f896ef3c8cbdcd56 Mon Sep 17 00:00:00 2001
From: Pablo Fraile Alonso <pablo@pablofraile.net>
Date: Mon, 26 Feb 2024 15:15:52 +0100
Subject: [PATCH] feat: print timeout on timeout timer update

---
 tcp/tcp.cc | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/tcp/tcp.cc b/tcp/tcp.cc
index a919c8c..d3de9d3 100644
--- a/tcp/tcp.cc
+++ b/tcp/tcp.cc
@@ -978,7 +978,10 @@ void TcpAgent::reset_rtx_timer(int mild, int backoff)
  */
 void TcpAgent::set_rtx_timer()
 {
-	rtx_timer_.resched(rtt_timeout());
+    double now = Scheduler::instance().clock()
+    double timeout = rtt_timeout();
+    printf("Time: %d Timeout %d\n", now, timeout);
+	rtx_timer_.resched(timeout);
 }
 
 /*
-- 
2.43.2

