From 385caef1628edb52c32378fedde06b76061d67e0 Mon Sep 17 00:00:00 2001
From: Pablo Fraile Alonso <pablo@pablofraile.net>
Date: Tue, 20 Feb 2024 18:00:09 +0100
Subject: [PATCH] feat: print timeout on reno

---
 tcp/tcp.cc | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/tcp/tcp.cc b/tcp/tcp.cc
index a919c8c..811e06e 100644
--- a/tcp/tcp.cc
+++ b/tcp/tcp.cc
@@ -978,7 +978,9 @@ void TcpAgent::reset_rtx_timer(int mild, int backoff)
  */
 void TcpAgent::set_rtx_timer()
 {
-	rtx_timer_.resched(rtt_timeout());
+    double timeout = rtt_timeout();
+    printf("Timeout for Reno is: %d", timeout);
+	rtx_timer_.resched(timeout);
 }
 
 /*
-- 
2.43.2

